docs/use_cases/otel/grafana.md

```diff
@@ -0,0 +1,148 @@
+# Grafana Integration
+
+Next step is to integrate with Grafana so that we can query Feldera for these insights.
```

Comment:
we don't query Feldera, supposedly Grafana does.
Explain what the flow is. 
I expected that Feldera would push changes to grafana, but perhaps you are querying a materialized view in Feldera?
(You could talk briefly about these two approaches)